modules/idxBidAdapter.js

```diff
@@ -0,0 +1,80 @@
+import { registerBidder } from '../src/adapters/bidderFactory.js'
+import { BANNER } from '../src/mediaTypes.js'
+import { isArray, isNumber } from '../src/utils.js'
+
+const BIDDER_CODE = 'idx_bidder_code'
+const ENDPOINT_URL = 'https://dev-event.dxmdp.com/rest/api/v1/bid'
+const SUPPORTED_MEDIA_TYPES = [ BANNER ]
+
+export const spec = {
+  code: BIDDER_CODE,
+  supportedMediaTypes: SUPPORTED_MEDIA_TYPES,
+  isBidRequestValid: function (bid) {
+    return isArray(bid.mediaTypes?.banner?.sizes) && !!bid.mediaTypes.banner.sizes.every(size => {
```

Comment:
Is the double negation needed here? I think it can be removed since `.every` returns a boolean value.